### PR TITLE
Add links to tutorials and general README clean up

### PR DESCRIPTION
**Pull request by dmey**
This PR adds links to documentation and tutorials as requested in #24 and tries to simplify the README.

NOTE: we may want to merge this in if/once the paper is accepted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,10 +79,40 @@ If you want to launch WRF-CMake and WPS-CMake built in `dmpar` to run on multipl
 - On Linux, use your package manager to download mpich (version ≥ 3.0.4). E.g. `sudo apt-get update && sudo apt-get install mpich` on Debian-based systems or `sudo yum install mpich` on RPM-based system like CentOS.
 
 
-### Source distribution
```
Comment (letmaik) [inline README.md:50]: Where did this go?

File: README.md [hunk 2]
```diff
@@ -47,10 +79,40 @@ If you want to launch WRF-CMake and WPS-CMake built in `dmpar` to run on multipl
...
-To build WRF-CMake from source, see [this page](doc/cmake/INSTALL.md).
+## Documentation
+
+- For the WRF model technical documentation, please refer to [A Description of the Advanced Research WRF Version 3](https://opensky.ucar.edu/islandora/object/technotes%3A500/datastream/PDF/view).
+- For the WRF model user documentation, please refer to [The Advanced Research WRF version 4 Modeling System User’s Guide](http://www2.mmm.ucar.edu/wrf/users/docs/user_guide_v4/contents.html).
+
+## Example usage
+
+If you have already used WRF/WPS before and you just want a quick tutorial to go over the main steps, we have put together a very basic tutorial on our sister-project's website [GIS4WRF](https://gis4wrf.github.io/) with step-by-step instructions [here](https://gis4wrf.github.io/tutorials/wrf-cmake/simulate-the-2018-european-heat-wave-with-wrf-cmake/).
```
Comment (letmaik) [inline README.md]: Please never use "here" as link word. It's too 90's.

File: README.md [hunk 3]
```diff
@@ -47,10 +79,40 @@ If you want to launch WRF-CMake and WPS-CMake built in `dmpar` to run on multipl
...
+
+Otherwise, if you are a complete beginner, we recommend going [trough the basics](http://www2.mmm.ucar.edu/wrf/OnLineTutorial/Basics/index.php) or [running the case studies](http://www2.mmm.ucar.edu/wrf/OnLineTutorial/CASES/index.php) as described in the [WRF-ARW Online Tutorial](http://www2.mmm.ucar.edu/wrf/OnLineTutorial/).
```
Comment (letmaik) [inline README.md]: Typo.

File: README.md [hunk 4]
```diff
@@ -27,7 +45,21 @@ WRF-CMake is designed to work alongside the current releases of WRF, therefore y
...
+The installation of WRF-CMake or WPS-CMake is straightforward thanks to the downloadable pre-built binaries for most Linux distributions (specifically [ RPM-based and Debian-based distribution-compatible](https://en.wikipedia.org/wiki/List_of_Linux_distributions)), macOS, and Windows (see [binary distribution](#binary-distribution-experimental) below) -- most users wishing to run WRF on their system can simply download the pre-compiled binaries without the need to build from source. Alternately, you can install WRF-CMake or WPS-CMake using the Homebrew/Linuxbrew package manager, or by building and installing the software from source -- please refer to the build and install [manually from source](#manually-from-source) and [using Homebrew or Linuxbrew](#using-homebrew-or-linuxbrew) section below.
+
+Please note that HPC users, or users seeking to run WRF in the 'most optimal' configuration for their system are advised to build WRF-CMake manually from source or to use the Homebrew/Linuxbrew package manager.
+
+
+### Manually from source
+To build and install WRF-CMake or WPS-CMake manually from source, see [this page](doc/cmake/INSTALL.md).
```
Comment (letmaik) [inline README.md]: Don't use "this page" as link target.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+Before creating a feature request, please check if a similar feature has been already asked [here](https://github.com/WRF-CMake/WRF/issues). If none exist please create a new feature request and include as many details as possible using the required template.
+
+## Coding conventions
+
+Same as in the [WRF](https://github.com/wrf-model/WRF) and [WPS](https://github.com/wrf-model/WPS).
+
+
+## Versioning
+
+WRF-CMake and WPS-CMake fully tracks releases made by the [WRF](https://github.com/wrf-model/WRF) and [WPS](https://github.com/wrf-model/WPS) model.
```
Comment (letmaik) [inline CONTRIBUTING.md]: WPS is not a model.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+Before creating bug reports, please check if similar issue have already been reported [here](https://github.com/WRF-CMake/WRF/issues). If none exist please create a new issue and include as many details as possible using the required template.
+
+## Request a new feature
+
+Before creating a feature request, please check if a similar feature has been already asked [here](https://github.com/WRF-CMake/WRF/issues). If none exist please create a new feature request and include as many details as possible using the required template.
+
+## Coding conventions
+
+Same as in the [WRF](https://github.com/wrf-model/WRF) and [WPS](https://github.com/wrf-model/WPS).
```
Comment (letmaik) [inline CONTRIBUTING.md]: missing word at the end

File: README.md (hunk 2 above)
Comment (dmey) [reply README.md]: Full path instead?

Comment (letmaik) [reply README.md]: No, either the title or something descriptive.

File: README.md (hunk 3 above)
Comment (dmey) [reply README.md]: Removed 'complete' in cb63872 but not sure about the typo.

File: README.md (hunk 2 above)
Comment (dmey) [reply README.md]: done in cb63872

File: README.md (hunk 4 above)
Comment (dmey) [reply README.md]: done in cb63872

File: CONTRIBUTING.md (hunk 5 above)
Comment (dmey) [reply CONTRIBUTING.md]: fixedin ef7032a

File: CONTRIBUTING.md (hunk 6 above)
Comment (dmey) [reply CONTRIBUTING.md]: fixedin ef7032a

File: README.md (hunk 3 above)
Comment (letmaik) [reply README.md]: trough -> through

**Review by zbeekman — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (dmey) [reply README.md]: Fixed in 3c357f4

**Pull request merged**